cympuser UI 수정 및 페이지 권한 관련 기능

kangisworking opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/nurigo-world/xe-module-cympuser/1)
<!-- Reviewable:end -->

Commit:
cympuser UI 수정 및 페이지 권한 관련 기능

wileyday commented:
스왑파일이 커밋되었음요!
.cympuser.admin.model.php.swp
.cympuser.admin.view.php.swp

wileyday commented on conf/module.xml:
```diff
@@ -1,7 +1,8 @@
 <module>
     <grants />
-    <permissions />
+	<permissions>
```
manager 퍼미션이 안들어 가 있는뎁쇼?

wileyday commented on cympuser.admin.controller.php:
```diff
@@ -59,5 +59,252 @@ function procCympuserAdminModDelete()
 		$this->setRedirectUrl($redirectUrl);
 	}
+
+	function procCympuserAdminAddProduct()
```
주석 달아주면 좋겠구먼!

wileyday commented on cympuser.admin.controller.php:
```diff
@@ -59,5 +59,252 @@ function procCympuserAdminModDelete()
+		$is_free = Context::get('is_free');
+		$item_srls = Context::get('item_srl');
+
+		if($target == 'elearning')
```
이 부분을 따로 메서드로 빼는건 어떠요?

wileyday commented on cympuser.admin.controller.php:
```diff
@@ -59,5 +59,252 @@ function procCympuserAdminModDelete()
+
+		}
+
+		if($target == 'nstore')
```
이 부분을 따로 메서드로 빼는건 어떠요?

wileyday commented:
메서드 procCympuserAdminAddProduct 에 if 구문을 통하여 복잡한 두가지 일을 하고 있음요!
리팩토링 필요한 듯~

wileyday commented on cympuser.admin.controller.php:
```diff
@@ -59,5 +59,252 @@ function procCympuserAdminModDelete()
+		$this->setRedirectUrl($success_return_url);
+	}
+
+	function getTotalPrice($item_srls)
```
주석요~

wileyday commented on cympuser.admin.controller.php:
```diff
@@ -59,5 +59,252 @@ function procCympuserAdminModDelete()
+		return $total_price;
+	}
+
+	function procCympuserAdminAddClassDays()
```
주석요

wileyday commented on cympuser.admin.model.php:
```diff
@@ -133,11 +132,40 @@ function getCympuserMemberList()
 	}
 
+
+	function getCympuserAdminItemList()
```
주석요

wileyday commented:
메서드 dispCympuserAdminMemberInsert 에 if 구문 통한 여러가지 일 하고 있음요!
이것도 리팩토링 필요하요~

wileyday commented on cympuser.admin.view.php:
```diff
@@ -205,10 +369,271 @@ function dispCympuserAdminMemberInsert()
+	 *
+	 * @return array
+	 */
+	function _getMemberInputTag($memberInfo, $isAdmin = false)
```
이건 다른데서 걍 가져온 듯.. 태클 걸지 않겠오

wileyday commented on lang/lang.xml:
```diff
@@ -1,171 +1,19 @@
-		<value xml:lang="ko"><![CDATA[대상모듈]]></value>
-		<value xml:lang="en"><![CDATA[Target Module]]></value>
+	<item name="payment_info">
+		<value xml:lang="ko"><![CDATA[결재정보]]></value>
```
결재가 아니라 결제임
결재정보 --> 결제정보

wileyday commented:
tpl/css/button.css
tpl/css/btn.css
tpl/css/classlist.css
tpl/img/icon_grid.gif
tpl/img/icon_list.gif

Commit:
Cympuser PR 에 대한 수정

Commit:
cympuser 기본정보 작업 중간 체크

Commit:
guidance_info 추가

Commit:
기본정보 / 학습관리 하면서 수정사항들

Commit:
컨테이너 100%로 변경해봄(리뷰 X)

Commit:
cympuser 회원 그룹별 필드 설정 기능
- 기존 XE에서는 모든 회원은 같은 설정을 따라야하는데 kpa 프로젝트를 하다보니 각 회원별 그룹에 따라 다른 설정이 필요하여 기능 추가함
- 추후 LMS 에도 꼭 필요한 기능 및 개선되어야할 기능인듯

Commit:
잡다한 정보 추가 및 수정 (리뷰 X)
